docs/ldml/tr35-general.html

```diff
@@ -2462,18 +2485,42 @@ <h3>6.4 <a name="perUnitPatterns" href="#perUnitPatterns" id=
+  …</pre>
+    </div>
     <p><strong>perUnitPattern</strong> — This is used as the
     denominator with another unit name. For example, a form such as
     "{0} per second" can be used to form "2 feet <strong>per
     second</strong>".</p>
     <p>The difference between these is that in some inflected
     languages, the compoundUnit cannot be used to form grammatical
     phrases. This is typically because the "per" + "second" combine
-    in a non-trivial way. For such languages, the compoundUnit
+    in a non-trivial way. </p>
+    <p>&nbsp;</p>
+    <p>For a description of how to use these fields to construct a localized name, see </p>
```

Comment:
see what? The link seems to be missing here

Reply:
fixed